<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetInfo</class>
 <widget class="QWidget" name="AssetInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>766</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TitleBar" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(57, 179, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>资产编号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="serialNum">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>资产名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="name">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>资产分类</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="classification">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>资产条码</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="qrCode">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>使用部门</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="department">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>存放地</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="storage">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>资产状态</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="state">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>规格</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="spec">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>采购日期</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="dateOfPurchase">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	background-color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>入库日期</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="dateOfStorage">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-color: rgb(99, 99, 99);
	border-width：2px solid;
	
	background-color: rgb(239, 239, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>396</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>widgets/TitleBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
